server/channels/app/file.go

```diff
@@ -1367,6 +1381,7 @@ func (a *App) CreateZipFileAndAddFiles(fileBackend filestore.FileBackend, fileDa
 	if err != nil {
 		return err
 	}
+
 	defer os.Remove(zipFileName)
```

Comment:
Nit pick: Unrelated change
```suggestion
	}
	defer os.Remove(zipFileName)
```